src/Formik.tsx

```diff
@@ -578,32 +582,47 @@ export function useFormik<Values extends FormikValues = FormikValues>({
   );
 
   const executeBlur = React.useCallback(
-    (e: any, path?: string) => {
-      if (e.persist) {
-        e.persist();
+    (eventOrValue: any | React.FocusEvent<any>, maybePath?: string) => {
```

Comment:
I changed `handleBlur` / `executeBlur` as well, since custom components may not to pass real blur event, but may pass value instead. Also I made pairs of functions `handleBlur`/`handleChange` and `executeBlur`/`executeChange` more similar (same type, same name of variables, etc)